.buildkite/generate-tags-pipeline.sh

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env nix-shell
+#! nix-shell shell.nix -i bash
+
+# SPDX-FileCopyrightText: 2022 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+set -euo pipefail
+
+ymlappend () {
+    echo "$1" >> .buildkite/pipeline-for-tags.yml
+}
+
+formulae=("tezos-accuser-011-PtHangz2" "tezos-admin-client" "tezos-baker-011-PtHangz2" "tezos-client" "tezos-codec" "tezos-endorser-011-PtHangz2" "tezos-node" "tezos-sandbox" "tezos-signer")
+
+ymlappend "
+ - label: Install tezos-sapling-params
+   key: install-tsp
+   agents:
+     queue: \"arm64-darwin\"
+   if: build.tag =~ /^v.*/
+   commands:
+   - brew install --formula ./Formula/tezos-sapling-params.rb"
+
+n=0
+for f in "${formulae[@]}"; do
+  n=$((n+1))
+  ymlappend "
+ - label: Build $f bottle for Big Sur arm64
+  key: build-bottle-$n
```

Comment:
My internal perfectionist is upset due to lack of space here and below :see_no_evil:  
```suggestion
   key: build-bottle-$n
```